logr.go

```diff
@@ -261,6 +277,10 @@ func (l Logger) WithName(name string) Logger {
 // If the underlying log implementation supports a WithCallDepth(int) method,
 // it will be called and the result returned.  If the implementation does not
 // support CallDepthLogSink, the original Logger will be returned.
+//
+// Helper() should be used instead of WithCallDepth(1) because Helper()
```

Comment:
Super nit - the `()` should be removed unless you are describing an actual call (e.g. `WithCallDepth(1)` is ok)